### ISSUE TITLE
[Grid][Accessibility] Filter Menu link does not have text

### ISSUE THREAD

**Issue by dtopalov** [telerik/kendo-angular#2138]
Grid Filter Menu link does not have text which causes an error in the Wave accessibility tool:

![image](https://user-images.githubusercontent.com/7533599/54750377-1777de80-4be0-11e9-8007-dbd4e04acd47.png)

**Comment by tsvetomir** [comment]
Merging with https://github.com/telerik/kendo-angular/issues/1598
